### DISCUSSION TITLE
Modrinth repositories are missing a lot of triage

### DISCUSSION THREAD

**Discussion by piprett** [modrinth/.github#276] [Other discussions]
TLDR; **Modrinth's various repositories are missing triage**.

There are a lot of issues that should be closed, but aren't. There are a lot of issues that should be transferred to other repositories. There are a lot of duplicate issues. There are a lot of issues with incorrect tags. There are a lot of invalid issues. There are discussions which should be converted to issues, and issues which should be converted to discussions. There are PRs that should be looked at, but haven't been.

Keeping your repositories tidy is a crucial to welcome contributors, and keep contributors to contribute. I personally don't feel encouraged to contribute, when it's messy everywhere. I find it to be a pain to find issues to work on, the issue trackers are littered with garbage. Issues that are out-of-scope, or won't be merged, are open. And even if I create PRs, I'm worried it will just sit there and collect dust for a few months without any activity.

I do however have big understanding of that your time is limited, and there are priorities. You are doing a lot of work, and we are quickly moving forward. Triaging issues might not be the most exciting either, we are developers after all. My suggestion to help aid the situation, is to give triage access to a small handful of volunteers[^1]. These volunteers will tag issues, close issues such as duplicates, and issues obviously not fit for the issue tracker (i.e modpack crashes). They will also transfer issues in wrong repositories, and convert discussions. Optionally they could also be allowed to merge certain PRs which are obviously good, like typos and dead links. This could also help users who believe certain things are bugs, but in reality isn't, as those will be redirected to the support portal, effectively gaining reputation for Modrinth. It might be hard to find good volunteers, but I believe this is not too big of an issue. Multiple people have already said they would be happy with having this role[^2][^3][^4]. Seeing how well modrinth/awesome[^1] went, I believe we can simply pick reputable members of the community.

[^1]: We can already see huge effect of adding a triage volunteer to the modrinth/awesome repository.
[^2]: https://discord.com/channels/734077874708938864/734077874708938867/1269720941068292207
[^3]: https://discord.com/channels/734077874708938864/734077874708938867/1269720978813091982
[^4]: https://discord.com/channels/734077874708938864/734077874708938867/1269724598673342486

**Comment by Minenash** [comment]
I've took the mantle for now. Tonight I've gone though roughly a third of the issues in the new mono repo, adding missing tags, removing some dublicates, and closing web issues that were already solved.

After I finish the other ⅔s, I'll go through the discussions (mainly moving bug posts)

**Comment by Minenash** [comment]
Only 5 more pages of the code repo issues
